.gitignore

```diff
@@ -114,3 +114,8 @@ data/catch-ignore.yml
 release_config.json
 web/userdata.js
 location.json
+
+#Multiple config
+configs/
+!configs/config.json.example
+!configs/release_config.json.example
```

Comment:
That's what this does. Ignore the config directory, but allow the two example files.